pom.xml

```diff
@@ -213,57 +157,6 @@
 						<appLocation>${project.artifactId}.war</appLocation>
 					</bootstrapProperties>
 				</configuration>
-				<executions>
-					<execution>
-						<id>install-liberty</id>
-						<phase>prepare-package</phase>
-						<goals>
-							<goal>install-server</goal>
-						</goals>
-					</execution>
-					<execution>
-						<id>package-app</id>
-						<phase>package</phase>
-						<goals>
-							<goal>package-server</goal>
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
-			<!-- Plugin to run functional tests -->
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-failsafe-plugin</artifactId>
-				<version>2.18.1</version>
```

Comment:
Basically same comments from the other reviews.  In the end make sure the test runs.